run_dir/design/worksets.html

```diff
@@ -0,0 +1,90 @@
+{% extends "base.html" %}
+
+<!-- 
+Template file: worksets.html
+URL: /worksets/[optional filter]
+Title: Summary of all Worksets
+Description: Shows a table with all worksets.
+-->
+
+{% block stuff %}
+
+{% if worksets == 'all' %}
+  <h1 id="{{ worksets }}" class="worksets_page_heading"><span id="page_title">All Worksets</span> <small>All worksets found in statusdb</small></h1>
+{% end %}
+
+
+<div id="loading_spinner" style="text-align:center; margin:100px 0;"><span class='glyphicon glyphicon-refresh glyphicon-spin'></span>  Loading worksets table..</div>
+<div id="page_content" style="display:none;">
+
+<form id="Search-form" class="form-inline marginBottom">
+  <div class="pull-right">
```

Comment:
As there are only two columns - unless more are planned, I'd remove this whole section and it's modal...